projects/ngx-avatar/src/lib/sources/google.ts

```diff
@@ -22,10 +22,12 @@ export class Google extends AsyncSource {
   /**
    * Extract google avatar from json data
    */
-  public processResponse(data: any, size?: number): string {
+  public processResponse(data: any, size?: number): string | null {
     const avatarSrc = data.entry.gphoto$thumbnail.$t;
     if (avatarSrc) {
       return avatarSrc.replace('s64', 's' + size);
     }
+
+    return null;
```

Comment:
Does compiler gives any error if we don't explicitly declare this line?

Reply:
Yes, it gives an error, because `undefined` (due to the lack of `return`) is not the same as `null`.

Reply:
Nice.